package.json

```diff
@@ -74,14 +74,14 @@
     "npm-shrinkwrap.json"
   ],
   "dependencies": {
-    "appium-adb": "^13.0.0",
-    "appium-android-driver": "^11.0.0",
+    "appium-adb": "^14.0.0",
```

Comment:
https://github.com/appium/appium-espresso-driver?tab=readme-ov-file#requirements needs to bump the min/recommended version